### PR TITLE
Add heap_size setting env for engine image

### PR DESCRIPTION
**Pull request by goldmedal**
# Description
Add two env to set the heap size of JVM.
- MAX_HEAP_SIZE
- INI_HEAP_SIZE

### CODE REVIEW

File: docker/entrypoint.sh [hunk 1]
```diff
@@ -1,6 +1,9 @@
+#!/bin/bash
+export ENV_MAX_HEAP_SIZE=$2
+export ENV_INI_HEAP_SIZE=$3
 python3 wren-sqlglot-server/main.py &
 
 # Required add-opens=java.nio=ALL-UNNAMED for Apache arrow in the Snowflake
-java -Dconfig=etc/config.properties \
+java -Xmx${ENV_MAX_HEAP_SIZE:-"512m"} -Xms${ENV_INI_HEAP_SIZE:-"64m"}  -Dconfig=etc/config.properties \
```
Comment (grieve54706) [inline docker/entrypoint.sh:7]: Curious, why use `INI` not `MIN`?

Comment (goldmedal) [reply docker/entrypoint.sh:7]: Initial heap size.

Comment (goldmedal) [reply docker/entrypoint.sh:7]: hmm.. maybe change to `MIN` would be more clear.

**Review by grieve54706 — APPROVED** [review]
LGTM

**Pull request merged**